# Accept and reject statistics, total and by round
uint32   total_accepted
uint32   total_rejected
uint32   round_accepted
uint32   round_rejected
